lib/modules/platform/bitbucket/pr-cache.ts

```diff
@@ -135,7 +137,13 @@ export class BitbucketPrCache {
       cacheProvider: repoCacheProvider,
     };
     const res = await http.getJson<PagedResult<PrResponse>>(url, opts);
-    this.reconcile(res.body.values);
+
+    const items = res.body.values;
+    logger.debug(`Fetched ${items.length} PRs to sync with cache`);
+
+    this.reconcile(items);
```

Comment:
If we have a faulty cache (somehow), and we are not self-healing through the reconcile command, would this logging be enough to catch it?